packages/mobile/src/send/utils.ts

```diff
@@ -1,18 +1,34 @@
 import BigNumber from 'bignumber.js'
-import { showError } from 'src/alert/actions'
+import { isLeft } from 'fp-ts/lib/Either'
```

Comment:
Do you know if this is tree shaken? https://bundlephobia.com/result?p=fp-ts@2.7.1

Reply:
this is used by utils and the attestations contract wrapper
doesn't seem like this is a significant regression